test/cloudtest/pkg/runners/shelltest_runner.go

```diff
@@ -51,14 +57,13 @@ func (runner *shellTestRunner) GetCmdLine() string {
 }
 
 // NewShellTestRunner - creates a new shell script test runner.
-func NewShellTestRunner(ids string, test *model.TestEntry, manager execmanager.ExecutionManager) TestRunner {
+func NewShellTestRunner(ids string, test *model.TestEntry, _ execmanager.ExecutionManager) TestRunner {
```

Comment:
Let's remove manager if it not used anythere.

Reply:
oops! good catch

Reply:
fixed